README.md

```diff
@@ -46,6 +46,26 @@ except Exception:
     logger.exception("Bad math.")
 ```
 
+###setup for Airbrake alternatives (e.g. Errbit)
+
+There are self-hosted alternatives to Airbrake which provide a compatible API (e.g. [Errbit](https://github.com/errbit/errbit)).
```

Comment:
@joegod - Another suggestion - maybe something about our onpremise offering?

"*Airbrake [Enteprise](https://airbrake.io/enterprise) and self-hosted alterantives, such as [Errbit](https://github.com/errbit/errbit), provide a compatible API.*"